index.Rmd

```diff
@@ -83,15 +83,15 @@ summary(fit)
 Show tables, plots, etc. and describe them.
 
 ```{r echo=TRUE, warning=FALSE}
-ggplotly (g, tooltip = c("County", "Opioid.Poisoning.Deaths", "Year"))
+ggplotly (g, tooltip = c("County", "Opioid.Poisoning.Deaths", "Year")) 
     
 ```
 
 ```{r echo=TRUE}
-mapview(vital_ny_2017, zcol = "Opioid.Poisoning.Deaths", legend = TRUE, alpha = 0.5)
+mapview(vital_ny_2017, zcol = "Opioid.Poisoning.Deaths", legend = TRUE, alpha = 0.5) 
 ```
 
-```{r echo=TRUE, warning=FALSE}
+```{r echo=TRUE, warning=FALSE} 
```

Comment:
Nice graph which show us the relationship between the death  and incomes. One problem is that when I first saw it, I did not know what "estimate" meant. I knew it after I read you introduction. So it could be better if you could make your labels more detailed and clear.
I could see there is the relationship between death and incomes. But I am curious why and expect that you could give us a more detailed description about your graph.